src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    class="ec-blog-item"
+  >
+    <div
+      class="tw-grid tw-border-b tw-border-solid tw-border-gray-6"
+    >
+      <div
+        class="tw-col-2"
+      >
+        <img
+          class="tw-w-96 tw-h-96 tw-rounded"
+          :src="featuredImage"
+        >
+      </div>
+      <div
+        class="tw-col-10"
+      >
+        <p
+          class="ec-blog-item__title"
+        >
+          {{ title }}
+        </p>
+        <p>
+          {{ author }}
+        </p>
+        <p>
+          {{ category }}
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EcBlogItem',
+  props: {
```

Comment:
I don't know witch prop must be required but I guess at least the title should be right? I didn't found it on the ticket.

A prop required should be declared like that:

`title: { 
  type: String,
  required: true,
}`